Updates all the circuit shells to share a palette

Wallemations opened the pull request:
## About The Pull Request
Before

![before](https://github.com/user-attachments/assets/47cba308-567b-47c3-8a1a-1a6b7ed979fb)

After (w/ `X`'s on the sprites I didn't touch)

![after](https://github.com/user-attachments/assets/3a3be1ef-65b1-409d-9f42-9290312c6c01)


I didn't touch:
Gun shell, USB cable, BCI
## Why It's Good For The Game
It was really bothering me how most of the shells had different greyscale palettes they were each using
## Changelog
:cl: Wallem
image: Resprited circuit shell icons
/:cl:

Wallemations committed:
yaay

Twaticus reviewed (approved):
assuming all the deleted icons are unused

Wallemations commented:
> assuming all the deleted icons are unused

Oh I could have sworn I added that to the desc

Yes, I removed all the unused ones and put them in a DMI file
I don't know where to put that so I just sent it into the discord's sprite channel
